apiary.apib

```diff
@@ -1403,6 +1403,13 @@ A workflow has the following attributes
 *id*, *created_at*, *updated_at*, *workflow_version*, *content_language*,
 and *classifications_count* are assigned by the API
 
+Three parameters: _grouped_, _prioritized_, and _pairwise_ configure
```

Comment:
Can you add a note about false by default means randon selection? Also should we metion pairwise priority sets? https://github.com/zooniverse/Cellect/blob/4b5341b8fb7dced6e9a8a6cda50cf6c804471595/lib/cellect/server/workflow.rb#L102